proto/test/v1/proto2/test_all_types.proto

```diff
@@ -31,21 +31,21 @@ message TestAllTypes {
   }
 
   // Singular
-  optional int32 single_int32 = 1 [default = -32];
-  optional int64 single_int64 = 2 [default = -64];
-  optional uint32 single_uint32 = 3 [default = 32];
-  optional uint64 single_uint64 = 4 [default = 64];
```

Comment:
I'm pretty certain there are tests that depend on the default values, please don't remove them.

Reply:
Whoops, that was unintentional. Copy paste got me. Fixed.